src/main/java/com/imaee/propinq/buildings/controllers/implementations/BuildingController.java

```diff
@@ -0,0 +1,27 @@
+package com.imaee.propinq.buildings.controllers.implementations;
+
+import com.imaee.propinq.buildings.controllers.interfaces.IBuildingController;
+import com.imaee.propinq.buildings.controllers.responses.BuildingResponse;
+import com.imaee.propinq.buildings.services.interfaces.IBuildingService;
+
+import java.util.List;
+import java.util.UUID;
+
+public class BuildingController implements IBuildingController {
+
+    private final IBuildingService buildingService;
+    public BuildingController(IBuildingService buildingService) {
+        this.buildingService = buildingService;
+    }
```

Comment:
Aprovechando que tenemos implementada la librería de Lombok, evitemos generar los constructores manualmente porque nos va a tomar tiempo de desarrollo si queremos agregar nuevos atributos a la inyección de dependencias.

```suggestion
@AllArgsConstructor
public class BuildingController implements IBuildingController {

    private final IBuildingService buildingService;
```

Y siempre dejá una línea de diferencia entre atributos y métodos. Para el orden viusal.